### ISSUE TITLE
Broken markdown table header

### ISSUE THREAD

**Issue by stsydow** [asyncapi/markdown-template#133]
https://github.com/asyncapi/markdown-template/blob/7797667f2c9cf58b8bfaa9473e13f2f2782a1031/components/common.js#L32
The column break needs three hyphens and not one:
```
  const breaks = `|${Array(headers.length).fill('---|').join('')}`;
```

**Comment by magicmatatjahu** [comment]
@stsydow Hello! Thanks for the issue. You're right, there must be at least 3 dashes separating each header cell, but markdown can be still valid in some environments like here, in Github. However... would you like to create a PR to fix this?

**Comment by magicmatatjahu** [comment]
Fixed in `@asyncapi/markdown-template@0.14.1`. Thanks for contribution!
